test/integration.sh

```diff
@@ -30,7 +30,9 @@ runTests() {
   fi
 
   go test -race $FLAGS github.com/kubernetes-incubator/service-catalog/test/integration/... -c \
-      && ./integration.test -test.v $@
+      && for test in $(./integration.test -test.list .); do
```

Comment:
I am ok with the hack fix here, but I think it would be valuable to track the issue and note it here so we can revert this change when the integration tests work as we want them to in the future.

Reply:
Something more than #1649 & #1660? I wasn't intending to close either of those based on this merging.